### PR TITLE
Playroom simulator bug fix

### PR DESCRIPTION
**Pull request by amburger66**
- robot should not be able to do some other action (e.g. picking a block, opening a door, toggling the dial) after moving to a new location

### CODE REVIEW

**Review by tomsilver — APPROVED** [review]
thanks! lgtm, but can you add a test or two that verifies that nothing happens if the robot tries to pick and move in the same timestep (as you described being disallowed in the PR comment)?

File: tests/envs/test_playroom.py [hunk 1]
```diff
@@ -213,6 +213,13 @@ def test_playroom_simulate_doors_and_dial():
     assert np.any(state[robot] != next_state[robot])
     assert np.any(state[door1] != next_state[door1])
     state = next_state
+    # Move to playroom door
+    act = Action(np.array([110.2, 15, 3, 1, 1]).astype(np.float32))
+    next_state = env.simulate(state, act)
+    for o in state:
+        if o.type != robot_type:
+            assert np.all(state[o] == next_state[o])
```
Comment (ronuchit) [inline tests/envs/test_playroom.py:221]: Might want to use allclose here for safety against precision issues? `np.allclose(state[o], next_state[o])`

Comment (ronuchit) [reply tests/envs/test_playroom.py:221]: Do you want to also add an `else: assert not np.allclose(state[o], next_state[o])`?

File: tests/envs/test_playroom.py [hunk 2]
```diff
@@ -222,6 +229,13 @@ def test_playroom_simulate_doors_and_dial():
     act = Action(np.array([105, 15, 3, 1, 1]).astype(np.float32))
     with pytest.raises(EnvironmentFailure):
         next_state = env.simulate(state, act)
+    # Move to dial
+    act = Action(np.array([125, 15.1, 1, -0.5, 1]).astype(np.float32))
+    next_state = env.simulate(state, act)
+    for o in state:
+        if o.type != robot_type:
+            assert np.all(state[o] == next_state[o])
```
Comment (ronuchit) [inline tests/envs/test_playroom.py:237]: same as above

File: tests/envs/test_playroom.py (hunk 1 above)
Comment (amburger66) [reply tests/envs/test_playroom.py:221]: good idea, thanks

**Comment by amburger66** [comment]
> thanks! lgtm, but can you add a test or two that verifies that nothing happens if the robot tries to pick and move in the same timestep (as you described being disallowed in the PR comment)?

the tests are already there! I will label them to be more clear about it

**Comment by tomsilver** [comment]
> > thanks! lgtm, but can you add a test or two that verifies that nothing happens if the robot tries to pick and move in the same timestep (as you described being disallowed in the PR comment)?
> 
> the tests are already there! I will label them to be more clear about it

oh just me being dumb, no need. thanks :)

**Pull request merged**